programs/eosc/main.cpp

```diff
@@ -160,13 +187,15 @@ int main( int argc, char** argv ) {
    string name;
    string ownerKey;
    string activeKey;
+   bool sign = false;
    auto createAccount = create->add_subcommand("account", "Create a new account on the blockchain", false);
    createAccount->add_option("creator", creator, "The name of the account creating the new account")->required();
    createAccount->add_option("name", name, "The name of the new account")->required();
    createAccount->add_option("OwnerKey", ownerKey, "The owner public key for the account")->required();
    createAccount->add_option("ActiveKey", activeKey, "The active public key for the account")->required();
+   createAccount->add_option("sign", sign, "Specify if unlocked wallet keys should be used to sign transaction");
```

Comment:
I really don't like how this add_option works for bool. It treats it as an int. So you have to pass 1 on the command line. But I just did what was done for the other existing bool options.

Reply:
Figured it out.` add_flag()` with "-s,--sign"